couchdb/source/pkg/apis/addtoscheme_serving_v1beta1.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package apis
 
 import (
-	servingv1beta1 "github.com/knative/serving/pkg/apis/serving/v1beta1"
+	servingv1beta1 "knative.dev/serving/pkg/apis/serving/v1beta1"
```

Comment:
that was the part of the failures

- we now have _only_  `knative.dev/serving` dependency (see also `Gopkg.lock`)